### DISCUSSION TITLE
Value Rain Yesterday for Wetterstation IP HmIP-SWO-PL

### DISCUSSION THREAD

**Discussion by Fhshaan** [OpenCCU/OpenCCU#2772] [🇺🇸 Q&A] [answered]
### Describe the solution you'd like

I want to integrate the rain amount yesterday in my programs.
Current RM relase 3.75.7.20240420 -
Wetterstation 1.2.6

### Describe alternatives you've considered

By Homematic?

### Is your feature request related to a problem?

jep - a practical one - my bonsais get too much water

**Comment by Baxxy13** [comment]
Die Daten gibt es schon auf der Zentrale.
Blende die systeminternen Systemvariablen ein und suche nach "rain" oder "yesterday"

**Comment by Fhshaan** [comment]
Ja, aber nicht als Wert in der „Programmierung“
Da kann man nur Regen aktueller Wert auswählen.

**Comment by Baxxy13** [comment]
Beim Gerät ja. 
Aber wenn du "Systemzustand" nimmst und auch dort wieder den Filter nutzt findest du die SysVar.

**Comment by Fhshaan** [comment]
Danke- hatte das nicht realisiert, da es bei der HM Wetterstation in dem Variablen war - und ich jetzt IP habe. 

Thx Fhs

**Comment by Baxxy13** [comment, marked as answer]
Um das hier nochmal zu präzisieren und ggf. zu schließen:
Der Sensor liefert einfach nur einen "ständig steigenden" Wert (wenn er Regen erkennt). 
Das steckt so in der Firmware des Gerätes und könnte höchstens von eQ-3 geändert/optimiert werden. 
Die ganzen Berechnungen werden durch WebUI-Programme realisiert und die Daten entsprechend in systeminterne Systemvariablen geschrieben. 
Auch wenn es eher wenig intuitiv ist kommt man an die Daten ran und somit braucht man keine zusätzlichen/doppelten Daten in der WebUI.

**Comment by Fhshaan** [comment]
Danke!!
Mittlerweile geklärt. Hatte vorher den HM Wettersensor und der hatte es in den Parametern, def IP wie du schreibst in den Sys-Var. 
Funktioniert jetzt. - thx fhs
